### PR TITLE
Fix bug in tREV

### PR DESCRIPTION
**Pull request by MaxBenChrist**

### CODE REVIEW

**Comment by MaxBenChrist** [comment]
I do not understand why the coverage decreases.

It could be that the an unrelated test, e.g. for the augmented_dickey_fuller throws an error non deterministically during the testing. Then the coverage drops and one line is less "tested".

File: tests/feature_extraction/test_feature_calculations.py [hunk 1]
```diff
@@ -462,11 +462,10 @@ def test_time_reversal_asymmetry_statistic(self):
         self.assertAlmostEqualOnAllArrayTypes(time_reversal_asymmetry_statistic, x, 0, 2)
         self.assertAlmostEqualOnAllArrayTypes(time_reversal_asymmetry_statistic, x, 0, 3)
 
-        x = [1, 2, -3, 4]
-        # 1/2 * ( (4^2 * 2 + 3 * 2^2) + (3^2*1)-(2*1^1)) = 1/2 * (32+12+9-2) = 51/2
-        self.assertAlmostEqualOnAllArrayTypes(time_reversal_asymmetry_statistic, x, 25.5, 1)
-        # 4^2 * 1 - 2 * 1^2 = 16 -2
-        self.assertAlmostEqualOnAllArrayTypes(time_reversal_asymmetry_statistic, x, 14, 2)
+        x = [1, 2, - 3, 4]
```
Comment (nils-braun) [inline tests/feature_extraction/test_feature_calculations.py]: whitespace :-D

File: tsfresh/feature_extraction/feature_calculators.py [hunk 2]
```diff
@@ -1066,12 +1066,11 @@ def time_reversal_asymmetry_statistic(x, lag):
     """
     n = len(x)
     x = np.asarray(x)
-    if 2 * lag > n:
+    if 2 * lag >= n:
         return 0
-    elif 2 * lag == n:
```
Comment (nils-braun) [inline tsfresh/feature_extraction/feature_calculators.py]: Can you explain why you have deleted this?

Comment (MaxBenChrist) [inline tsfresh/feature_extraction/feature_calculators.py]: if you have a time series of length `n`, you can only lag it `n-1`.

Here this is means, that the feature is not defined for 2 * lag == n.

**Comment by MaxBenChrist** [comment]
fine with merging @nils-braun ?

**Pull request merged**